tools/cmake/afr_metadata.cmake

```diff
@@ -358,6 +364,15 @@ function(afr_write_metadata)
     )
 endfunction()
 
+# This function is used for any additional files that need
+# be included into the OCW build combo
+function(afr_files_to_console_metadata)
```

Comment:
If this function is called multiple times in the same build combination, wouldn't the variable be reset?
It may be better to update this function to support appending to the `AFR_FILES_TO_CONSOLE_METADATA` list if called multiple times.